src/main.rs

```diff
@@ -1,6 +1,7 @@
 /************************************************************************
  *                                                                      *
  * Copyright 2014-2015 Urban Hafner, Thomas Poinsot                     *
+ * Copyright 2015 Igor Polyakov                                         *
```

Comment:
Sorry to be so anal, but could you please split the 2014-2015 line and put all of us three into the 2015 line?